setup.py

```diff
@@ -28,19 +29,28 @@
     )
 ]
 
-install_requires=[
-    "triton>=2.1.0",
-    "stanford-stk==0.7.0",
+install_requires = [
+    'numpy>=1.21.5,<2.1.0',
+    'torch>=2.3.0,<2.4',
+    'triton>=2.1.0',
+    'stanford-stk @ git+https://git@github.com/stanford-futuredata/stk.git@a1ddf98466730b88a2988860a9d8000fd1833301',
+    'packaging>=21.3.0,<24.2',
 ]
 
 extra_deps = {}
 
 extra_deps["gg"] = [
-    "grouped_gemm==0.1.4",
+    'grouped_gemm @ git+https://git@github.com/tgale96/grouped_gemm.git@66c7195e35e8c4f22fa6a014037ef511bfa397cb',
```

Comment:
@tgale96 needs to cut a new release. The last pinned release breaks because of the build in isolation error. Again, we should talk to Trevor about this.

In the meantime though, I just pinned to the last commit. I think we should merge with this and update when @tgale96 makes the new release.